test_tracetools/test/test_buffer.py

```diff
@@ -56,32 +56,31 @@ def test_all(self):
         # Check corresponding events for construct_buffer_event
         for construct_event in construct_buffer_events:
             target_buffer = self.get_field(construct_event, 'buffer')
-            target_buffer_to_ipb_event = self.get_events_with_field_value(
+            # Only 1 for our given buffer
```

Comment:
Can we delete this comment, since it is obvious with `allow_multiple=False` argument? and else where.

Reply:
good idea, see 8f97dedc3c9241db443d3e5936817a2aed6c9bf8. I kept some comments that explained _why_ there should be only 1 event, but removed the ones that basically just said that there's only 1 event.